webapp/channels/src/components/drafts/drafts.scss

```diff
@@ -36,6 +37,41 @@
         margin-bottom: 20px;
     }
 
+    #draft_tabs {
+        .nav.nav-tabs {
+            .drafts_tab {
+                .badge {
+                    position: relative;
+                    top: -2px;
+                    display: inline-block;
+                    height: 14px;
+                    margin-left: 6px;
+                    background: rgba(var(--center-channel-color-rgb), 0.2);
+                    color: rgba(var(--center-channel-color-rgb), 0.75);
+                    font-size: 11px;
+                }
+
+                &.active {
+                    .badge {
+                        background: rgba(var(--button-bg-rgb), 0.08);
+                        color: var(--button-bg);
+                    }
+                }
+            }
+        }
+
+        .tab-content {
+            height: 100%;
+
+            div[role="tabpanel"] {
+                position: relative;
+                height: 100%;
+            }
+        }
+    }
+
+    span.MuiBadge-badge {position: relative;top: -2px;}
```

Comment:
wonder why the linting didnt catch this one

Reply:
Done.